.circleci/config.yml

```diff
@@ -149,28 +155,20 @@ workflows:
   version: 2
   build:
     jobs:
-      - initial_checks
-      - tests_py36
-      - tests_py37
-      - tests_py38
-      - tests_py39_coverage_docs
+      - initial checks
+      - test against python 3_6 and git 1_8_10
+      - test against python 3_7 and git 2_7_6
+      - test against python 3_8 and git 2_25_0 with PEP8 checks
+      - test against python 3_9 and git 2_30_1 with coverage report and docs creation
```

Comment:
this needs to stay, as yaml cannot extend arrays, here is further explanation:
https://github.com/yaml/yaml/issues/35

Reply:
Wow didn't know about this limitation 😯